website_mail_snippet_bg_color/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================================
+Email Snippets Background Color Chooser
+=======================================
+
+This module extends the functionality of the website mail designer to support
+setting a custom background color to any snippet and allow you to customize it.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Install any module that makes use of the website mail designer, such as
+   ``mass_mailing``.
+#. Edit an email with the website mail designer.
+#. Drag & drop any snippet into the mail body.
+#. Choose *Customize > Choose Background Color*.
+#. Pick any color, or enter its HTML code in the box, or press *Clear* to
```

Comment:
Pick any color or enter its HTML code in the box. Press *Clear* to
   remove it.